recipes/qr-code-generator/all/CMakeLists.txt

```
@@ -0,0 +1,44 @@
+﻿cmake_minimum_required (VERSION 2.8.12)
```

If I remember, `set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)` requires CMake 3.4 at least
```suggestion
﻿cmake_minimum_required (VERSION 3.4)
```